src/serac/physics/thermomechanics.hpp

```diff
@@ -156,15 +186,11 @@ class Thermomechanics : public BasePhysics {
    */
   void advanceTimestep(double& dt) override
   {
-    if (coupling_ == serac::CouplingScheme::OperatorSplit) {
-      double initial_dt = dt;
-      thermal_.advanceTimestep(dt);
-      solid_.advanceTimestep(dt);
-      SLIC_ERROR_ROOT_IF(std::abs(dt - initial_dt) > 1.0e-6,
-                         "Operator split coupled solvers cannot adaptively change the timestep");
-    } else {
-      SLIC_ERROR_ROOT("Only operator split coupling is currently implemented");
-    }
+    double initial_dt = dt;
+    thermal_.advanceTimestep(dt);
+    solid_.advanceTimestep(dt);
+    SLIC_ERROR_ROOT_IF(std::abs(dt - initial_dt) > 1.0e-6,
+                       "Operator split coupled solvers cannot adaptively change the timestep");
```

Comment:
Are we always operator split coupled now? (whatever that means)

Reply:
Yes - the complexities of handling operator split and monolithic solves in one module would be a pretty big challenge. If we decide to go down that route, it will be a separate (big) PR.